code/game/machinery/suit_storage_unit.dm

```diff
@@ -131,54 +351,49 @@
 	if(stat & NOPOWER)
 		return
 	if(!user.IsAdvancedToolUser())
-		return 0
-	if(src.panelopen) //The maintenance panel is open. Time for some shady stuff
+		return FALSE
+	if(panel_open) //The maintenance panel is open. Time for some shady stuff

```

Comment:
There's no way to access this anymore, it used to be using a screwdriver. I think the object damage PR broke this a while ago, but it should be fixed.